src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1491,6 +1491,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return typesXmlConverter;
```

Comment:
@iSazonov, your last commit had 67 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

TypeTable duplicate types in reused runspace InitialSessionState TypeTable.No duplicate types test.Error occurred in Context block
```powershell
Multiple ambiguous overloads found for "Add" and the argument count: "1".
At D:\a\1\s\test\powershell\engine\Api\InitialSessionState.Tests.ps1:72 char:13
```
TypeTable duplicate types in reused runspace InitialSessionState TypeTable.Cannot use shared TypeTable in ISS test.Error occurred in Context block
```powershell
Multiple ambiguous overloads found for "Add" and the argument count: "1".
At D:\a\1\s\test\powershell\engine\Api\InitialSessionState.Tests.ps1:105 char:13
```
ConsoleHost unit tests.Clear-Host does not injects data into PowerShell output stream
```powershell
Multiple ambiguous overloads found for "SetBufferContents" and the argument count: "2".
at <ScriptBlock>, : line 4
4: $RawUI.SetBufferContents(

```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable -All gets all compatible modules when SkipEditionCheck: $true, using root module: $true, using absolute path: $false, CompatiblePSEditions:
```powershell
Cannot convert argument "value", with value: "c63616fa-596a-4fc3-a550-e5da822d80a2", for "Contains" to type "System.Char": "Cannot convert the "c63616fa-596a-4fc3-a550-e5da822d80a2" value of type "System.Guid" to type "System.Char"."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 534
534:                 Get-Module -ListAvailable -All -SkipEditionCheck | Where-Object { $_.Path.Contains($guid) }

```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable -All gets all compatible modules when SkipEditionCheck: $true, using root module: $true, using absolute path: $false, CompatiblePSEditions:
```powershell
Cannot convert argument "value", with value: "b353e065-cc0d-42db-89c5-1dd1bea32683", for "Contains" to type "System.Char": "Cannot convert the "b353e065-cc0d-42db-89c5-1dd1bea32683" value of type "System.Guid" to type "System.Char"."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 534
534:                 Get-Module -ListAvailable -All -SkipEditionCheck | Where-Object { $_.Path.Contains($guid) }

```